runtime/testing-hydradx/src/lib.rs

```diff
@@ -420,19 +620,117 @@ impl EnsureOrigin<Origin> for EnsureRootOrTreasury {
 	}
 }
 
-impl pallet_relaychain_info::Config for Runtime {
+impl orml_vesting::Config for Runtime {
 	type Event = Event;
-	type RelaychainBlockNumberProvider = RelayChainBlockNumberProvider<Runtime>;
+	type Currency = Balances;
+	type MinVestedTransfer = MinVestedTransfer;
+	type VestedTransferOrigin = GalacticCouncilOrRoot;
+	type WeightInfo = ();
+	type MaxVestingSchedules = MaxVestingSchedules;
+	type BlockNumberProvider = System;
 }
 
-impl pallet_preimage::Config for Runtime {
+impl InstanceFilter<Call> for ProxyType {
```

Comment:
common runtime?

Reply:
the same problem as with `OriginCaller`, `Call` enum is constructed in `lib.rs` and not accessible in common runtime.